runtimes/talos-libgdx/src/com/talosvfx/talos/runtime/modules/TextureModule.java

```diff
@@ -47,7 +47,8 @@ public void processValues() {
 
     public void setRegion (String regionName, Sprite region) {
         this.regionName = regionName;
-        if(region != null) {
+        
```

Comment:
this came with @wtflian's changes, unrelated to this fix. but the heck. let's let it be.